server/lists/adapters/workstacks.js

```diff
@@ -42,6 +42,29 @@ const defaultCaseSort = (a, b) => {
     }
     return sortResult;
 };
+
+const byTag = (a, b) => {
```

Comment:
I had to look at this for a while to understand what it's doing but am not sure why it is doing it. Seems like a very specific version of sorting based on length. Maybe needs more explanation through variable names or comments?